src/core/Utils.js

```diff
@@ -122,13 +122,15 @@ function getFileType (file) {
     'mp3': 'audio/mp3'
   }
 
-  // no smart detection for remote files, just trust the provider
+  const fileExtension = getFileNameAndExtension(file.name)[1]
+
   if (file.isRemote) {
-    return Promise.resolve(file.type.split('/'))
+    // some providers do not support for file types
+    let mime = file.type ? file.type : extensionsToMime[fileExtension]
+    const type = mime ? mime.split('/') : emptyFileType
+    return Promise.resolve(type)
```

Comment:
Was going to suggest this when I read the mime type comment below, but you already did it :D